src/analysis/measure_arguments.Rmd

```diff
@@ -62,8 +47,11 @@ measurement_codename = "50_1"
 
 result_geth = load_data_set("geth", program_set_codename, measurement_codename)
 result_evmone = load_data_set("evmone", program_set_codename, measurement_codename)
-result_openethereum = load_data_set("openethereum", program_set_codename, "400_4")
+# was originally 400_4, but this goes over the memory, must reduce to less
+result_openethereum = load_data_set("openethereum", program_set_codename, "200_2")
 results = rbind(result_evmone, result_geth, result_openethereum)
+# TODO geth short-circuits zero length programs, resulting in zero timing somehow. Drop these more elegantly, not based on measure_total_time_ns
```

Comment:
how did we get zero length programs?

Reply:
JUMPDEST for `op_count == 0`. It is nullary and has no returns, so the generator spits out an empty program. For `op_count != 0`, the program is just a sequence of `JUMPDEST`s. We'll have to figure out how to treat JUMPDEST fairly.